<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowController</class>
 <widget class="QMainWindow" name="MainWindowController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Toggl Track</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="ErrorViewController" name="widget" native="true"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="OverlayWidget" name="overlayWidget"/>
      <widget class="LoginWidget" name="loginWidget"/>
      <widget class="TimeEntryEditorWidget" name="timeEntryEditorWidget"/>
      <widget class="TimeEntryListWidget" name="timeEntryListWidget"/>
      <widget class="IdleNotificationWidget" name="idleNotificationWidget"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuToggl_Desktop">
    <property name="title">
     <string>To&amp;ggl Track</string>
    </property>
    <addaction name="actionEmail"/>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionContinue"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionShow"/>
    <addaction name="separator"/>
    <addaction name="actionSync"/>
    <addaction name="actionReports"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionSend_Feedback"/>
    <addaction name="actionHelp"/>
    <addaction name="actionLogout"/>
    <addaction name="actionClear_Cache"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuToggl_Desktop"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="text">
    <string>&amp;Continue</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="text">
    <string>S&amp;ync</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionReports">
   <property name="text">
    <string>&amp;Reports</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSend_Feedback">
   <property name="text">
    <string>Send &amp;Feedback</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>&amp;Logout</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionClear_Cache">
   <property name="text">
    <string>Clear Cache</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionEmail">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;email</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>Sh&amp;ow</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ErrorViewController</class>
   <extends>QWidget</extends>
   <header>errorviewcontroller.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OverlayWidget</class>
   <extends>QWidget</extends>
   <header>overlaywidget.h</header>
  </customwidget>
  <customwidget>
   <class>LoginWidget</class>
   <extends>QWidget</extends>
   <header>loginwidget.h</header>
  </customwidget>
  <customwidget>
   <class>TimeEntryEditorWidget</class>
   <extends>QWidget</extends>
   <header>timeentryeditorwidget.h</header>
  </customwidget>
  <customwidget>
   <class>TimeEntryListWidget</class>
   <extends>QWidget</extends>
   <header>timeentrylistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>IdleNotificationWidget</class>
   <extends>QWidget</extends>
   <header>idlenotificationwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
